jina/peapods/pea.py

```diff
@@ -415,7 +415,9 @@ def close(self) -> None:
         self.send_terminate_signal()
         self.is_shutdown.wait()
         if not self.daemon:
+            # TODO: remove clear_queues when queues are totally removed
```

Comment:
Should this TODO stay or be removed before merging the PR?

Reply:
no, because it is a TODO for the future